proggy.cpp

```diff
@@ -76,14 +111,20 @@ std::vector<Customer> loadCustomers(std::string fname) {
     std::ifstream f(fname);
     nlohmann::json j;
 
+//** KOOS:
+//** This if statement does nothing - i.e. if (f) was !true, nothing is done about it..
```

Comment:
Yes and no -- it'll raise a different error in this case if f isn't a valid file. This could be handled much better in terms of error checking.